src/Globe/Globe.js

```diff
@@ -55,15 +54,15 @@ function Globe(ellipsoid, gLDebug) {
     this.tiles = new Quadtree(TileMesh, this.SchemeTileWMTS(schemeTile_1), kml);
     this.layersConfiguration = new LayersConfiguration();
 
-    this.atmosphere = this.NOIE ? new Atmosphere(this.ellipsoid) : undefined;
+    this.atmosphere = this.NOIE ? new Atmosphere(ellipsoid) : undefined;
     this.clouds = new Clouds();
 
     var material = new BasicMaterial(new THREE.Color(1, 0, 0));
 
     var geometry = new THREE.SphereGeometry(5);
     var batiment = new THREE.Mesh(geometry, material);
 
-    var position = this.ellipsoid.cartographicToCartesian(new GeoCoordinate(0, 48.87, 200, UNIT.DEGREE));
+    var position = new Coordinates('EPSG:4326', 0, 48.87, 200).as('EPSG:4328');
```

Comment:
Yes it does (well I can even remove this line since `position` is overwritten in the next line)